<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>E3t4</class>
 <widget class="QWidget" name="E3t4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="vecWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="vecWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="vecWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>360</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="vecWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>220</y>
     <width>201</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>vecWidget</class>
   <extends>QWidget</extends>
   <header>vecwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
